source/fundamentals/indexes.txt

```diff
@@ -200,7 +200,7 @@ The following is an example of a query that would be covered by the index create
 
 .. common-content-begin
 
-See the MongoDB server manual section on :manual:`Compound indexes </core/index-compound>` for more information.
+See the MongoDB Server manual section on :manual:`Compound indexes </core/index-compound>` for more information.
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".